data/battle_scripts_1.s

```diff
@@ -5841,8 +5841,26 @@ BattleScript_LearnedNewMove::
 BattleScript_LearnMoveReturn::
 	return
 
+@@@ === NEW WEATHER SCRIPT
+
+BattleScript_WeatherContinues::
+	printfromtable gWeatherTurnStringIds
+	waitmessage B_WAIT_TIME_LONG
+	playanimation_var BS_ATTACKER, sB_ANIM_ARG1
+	setbyte gBattleCommunication, 0
+	call BattleScript_ActivateWeatherAbilities
+	end2
+
+BattleScript_WeatherFaded::
+	printfromtable gWeatherEndsStringIds
+	waitmessage B_WAIT_TIME_LONG
+	call BattleScript_ActivateWeatherAbilities
+	end2
+
+@@@ === NEW WEATHER SCRIPT
+
 BattleScript_RainContinuesOrEnds::
-	printfromtable gRainContinuesStringIds
+	@printfromtable gRainContinuesStringIds
```

Comment:
Why are these commented out?

Reply:
those scripts are unused. I just commented stuff out that I already deleted but I didn't want to do any further clean up.